Package/scripts/postinstall

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+exec "$DSTROOT/describealign.app/Contents/MacOS/describealign" --install-ffmpeg
```

Comment:
With the mac installer, the files end up being owned by root as they are installed in /Applications/describealign.app, so if it tries to install on first run it will fail with permissions errors. `postinstall` gets run in the context of the installer which ensures it has the proper permissions to write to those paths